src/executor/graph_executor.cc

```diff
@@ -1018,20 +1155,37 @@ void GraphExecutor::Init(nnvm::Symbol symbol,
                                 g.GetAttr<StorageTypeVector>("storage_type"));
   }
 
+  if (dmlc::GetEnv("MXNET_USE_TENSORRT", false)) {
```

Comment:
can we lookup the env var once and save it as a local variable like https://github.com/apache/incubator-mxnet/blob/master/src/executor/graph_executor.cc#L41?

Reply:
@eric-haibin-lin @KellenSunderland The reason this is not being cached the way `log_verbose_` is, is mostly because users can change the value of the env var at runtime more than once. An example of such a situation can be found [here](https://github.com/mkolod/incubator-mxnet/blob/tensorrt_integration/tests/python/tensorrt/test_tensorrt_lenet5.py#L77::L85) in the unit tests. The reason for this being an env var was to not break any of the C APIs that are tied to all language frontends. Allowing the check at simple_bind time each time allows to run with or without TensorRT, to compare performance, for instance. Note that this env var only gets checked twice in two different methods during bind time, while `log_verbose_` gets checked 9 times. Note that once a symbol is bound, the env var no longer gets checked. It's not common for a symbol to be rebound, and even when that happens, it's just during setup, not anwhere else later on.

Reply:
@eric-haibin-lin Does the above satisfy you as to why it would be preferable to not make the value static?

Reply:
Just to clarify I didn't mean to make it static, but rather to cache the value in the Executor constructor. The `bool log_verbose_ ` variable is a member variable only to that executor, so you can set env var at runtime to get different executor if you call simple_bind twice on the same symbol with and without TRT. That being that, it's a minor improvement suggestion since it's not being repeatedly looked up.

Reply:
@eric-haibin-lin Thanks for the clarification!